cli/maybe.ts

```
@@ -0,0 +1,7 @@
+export function isJust<T>(maybe: Maybe<T>): boolean {
+  return maybe.constructor.name === "Just";
+}
+
+export function fromJust<T>(maybe: Maybe<T>): T {
+  return maybe.value0;
```

Do you want to throw an exception if this is invalid?